matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/permalinks/PermalinkService.kt

```diff
@@ -32,40 +33,45 @@ interface PermalinkService {
      * Ex: "https://matrix.to/#/!nbzmcXAqpxBXjAdgoX:matrix.org/$1531497316352799BevdV:matrix.org"
      *
      * @param event the event
+     * @param forceMatrixTo whether we should force using matrix.to base URL
      *
      * @return the permalink, or null in case of error
      */
-    fun createPermalink(event: Event): String?
+    fun createPermalink(event: Event, forceMatrixTo: Boolean = false): String?
```

Comment:
We try to avoid default param on SDK, but I guess it's ok?

Reply:
Default behaviour is "do not force matrix.to", Benoit asked me to add this param but I'm not sure we really need it, or maybe a single method `createMatrixToPermalink(event: Event)` would be good enough?